powerfulseal/policy/policy_runner.py

```diff
@@ -58,42 +59,37 @@ def run(cls, policy, inventory, k8s_inventory, driver, executor,
             metric_collector=None):
         """ Runs a policy forever
         """
-        config = policy.get("config", {})
+        config = policy.get("config", {}).get("runStrategy", {})
+        should_randomize = config.get("strategy") == "random"
         wait_min = config.get("minSecondsBetweenRuns", 0)
         wait_max = config.get("maxSecondsBetweenRuns", 300)
-        loops = config.get("loopsNumber", None)
-        node_scenarios = [
-            NodeScenario(
-                name=item.get("name"),
-                schema=item,
-                inventory=inventory,
-                driver=driver,
-                executor=executor,
-                metric_collector=metric_collector
-            )
-            for item in policy.get("nodeScenarios", [])
-        ]
-        pod_scenarios = [
-            PodScenario(
+        loops = config.get("runs", None)
+        scenarios = [
+            Scenario(
                 name=item.get("name"),
                 schema=item,
                 inventory=inventory,
                 k8s_inventory=k8s_inventory,
+                driver=driver,
                 executor=executor,
                 metric_collector=metric_collector
             )
-            for item in policy.get("podScenarios", [])
+            for item in policy.get("scenarios", [])
         ]
         while loops is None or loops > 0:
-            for scenario in node_scenarios:
-                scenario.execute()
-            for scenario in pod_scenarios:
-                scenario.execute()
-            sleep_time = int(random.uniform(wait_min, wait_max))
-            logger.info("Sleeping for %s seconds", sleep_time)
-            time.sleep(sleep_time)
+            if len(scenarios) == 0:
```

Comment:
Ultra nit. Using the implicit booleanness of the empty list is more Pythonic [[0](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)]